functions.php

```diff
@@ -160,6 +159,18 @@ function siteorigin_corp_scripts() {
 
 	// Theme JavaScript.
 	wp_enqueue_script( 'siteorigin-corp-script', get_template_directory_uri() . '/js/jquery.theme' . SITEORIGIN_THEME_JS_PREFIX . '.js', array( 'jquery' ), SITEORIGIN_THEME_VERSION, true );
+	
+	// Mobile menu collapse localisation.
+	if ( siteorigin_setting( 'navigation_mobile_menu_collapse' ) != 768 ) {
```

Comment:
A few small style things here. First we're using the PHP var `$collapse_array`, which is being assigned to the JS var `siteorigin_corp_resp_menu_params`. It'd be neater to call the PHP variable $menu_params or something.

Second, why have this conditional? It's basically saying if the setting is not equal to 768, then output the value, or else output 768. Why not just always output `siteorigin_setting( 'navigation_mobile_menu_collapse' )`, maybe passed through `intval` to be safe.

Reply:
The purpose of this conditional is so that users can add a new body class using Code Snippets plugin and thereby disable the sticky header on mobile.

Reply:
`if ( ! function_exists( 'siteorigin_corp_child_body_classes' ) ) :
function 'siteorigin_corp_child_body_classes( $classes ) {
	$classes[] = 'mobile-header-ns';
  	return $classes;
}
endif;
add_filter( 'body_class', ''siteorigin_corp_child_body_classes' );`

Reply:
Will make changes as suggested. Big love.